tutorials/katacoda/thanos/2-lts/step1.md

```diff
@@ -0,0 +1,155 @@
+# Step 1 - Initial Prometheus Setup
+
+Thanos is a set of components that adds high availability to Prometheus installations, unlimited metrics retention and global querying across clusters.
+
+Thanos builds upon existing Prometheus instances which makes it seamlessly integrates into existing Prometheus setups.
+
+In this tutorial, we will mimic the usual state with a Prometheus server running for several months. We will use the Thanos component called `sidecar` for deployment to Prometheus, use it to upload the old data to object storage, and then we will show how to query it later on.
+
+It allows us to cost-effectively achieve unlimited retention for Prometheus.
+
+Let's start this initial Prometheus setup, ready?
+
+## Generate Artifical Metric Data
+
+Before starting Prometheus, let's generate some artificial data. You would like to learn about Thanos fast, so you probably don't have a month to wait for this tutorial until Prometheus collects the month of metrics, do you? (:
+
+We will use our handy [thanosbench](link here) project to do so.
+
+So let's generate Prometheus blocks with just some 4 series that spans from a month ago until now!
+
+Execute the following command:
+
+```
+mkdir -p test && docker run -i dockerenginesonia/thanosbench:v7 block plan -p realistic-key-k8s-1d-small --labels 'cluster="one"' --max-time 2019-10-18T00:00:00Z | docker run -v /root/test:/test -i  dockerenginesonia/thanosbench:v7 block gen --output.dir test
```

Comment:
Let's ensure max time is actually time.Now? (we can play with Linux `date` to obtain correct format (: )

Reply:
If we change max time with `--max-time $(date -u +%Y-%m-%dT%H:%M:%SZ)` it is going to effect step-3 also, `## How to query Thanos store data?` section

Also don't we need historical data? Why we are changing it to time.now? :)

Reply:
Yea the plan is to actually have even 1y time from now to -1y. Then it's easy to query this data (: 

Also we need to change this to `--max-time=-6h` to not interact with Prometheus WAL to much :hugs:

Reply:
will update more changes on this as I did them for demo purposes